src/app/models/resource-language.ts

```diff
@@ -10,4 +10,5 @@ export class ResourceLanguage {
   language: Language;
   customPages: CustomPage[];
   customTips: CustomTip[];
+  'attr-include-tips': 'true' | 'false';
```

Comment:
didn't realize that this syntax existed for attributes with `-` in the name

Reply:
Ya object initialization works with just strings. But accessing it requires the indexing syntax `object['some-key']`.

Also in ES6 you can use a variable as the left hand side as the key like this:
```ts
const keyName = 'testKey';

expect({
  [keyName]: 1,
}).toEqual({
  testKey: 1,
});